csuntechlab/checkpoint
# Check 64 - Create set up script for dev onboarding

# Description
- I've changed the `start.sh` to find a Windows machine's primary IP under WSL. 
- Other changes include:  `docker-compose` bootstraps the project efficiently, exposes mysql for local connections with mysql clients, and `.dockerignore` for production builds (Future use case)
- This new change also requires a documentation update when it's merged. 
- Removes `./start.sh up -d` functionality

## Type of change

# How Has This Been Tested?
 ⚠️**I need someone on the team with a Windows computer to test this our under WSL.** ⚠️
- Copy the `.env.example` file as `.env`
- Run `./start.sh` in your terminal, verify script output and `.env file changes`

Docker changes (Anyone can test)
- Remove `vendor/` directory
- Run `docker-compose up -d --build` and visit [localhost:8080](http//localhost:8080) when the build is complete
  - Also verify composer packages are installed in the `vendor/` directory
- Use Sequel Pro or your favorite MySql client and connect to the checkpoint database

# Checklist:
